app/views/pets/show.html.erb

```diff
@@ -7,6 +7,10 @@
 <p>Shelter: <%= link_to "#{@pet.shelter.name}", "/shelters/#{@pet.shelter.id}" %></p><br>
 <p>Status: <%= @pet.adoption_status %></p><br>
 
-<%= link_to 'Favorite', "/favorite/#{params[:id]}", method: :patch %><br>
+  <% if favorites.pet_check(params[:id]) == false%>
```

Comment:
Mike said we shouldn't use params in our views. I know I did for the first part. We can fix now or refactor later.

Reply:
I think refactoring later is the best idea, lets focus on getting everything working first